common/changes/@microsoft/api-extractor/octogonz-ae-issue1261_2019-05-05-19-26.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Add a new setting \"omitTrimmingComments\" to prevent extra comments from being emitted in the .d.ts rollup",
+      "type": "patch"
```

Comment:
Minor?

Reply:
This feature is pretty trivial. I don't think it merits a minor bump.